index.html

```diff
@@ -1,74 +1,155 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
```

Comment:
w materiałach CC było info że nie trzeba zamykać takich znaczników. to zrobił Ci auto formatter?

Reply:
tak, nic tu nie ruszałam